modern/src/runtime/PopupMenu.js

```diff
@@ -2,6 +2,13 @@ import GuiObject from "./GuiObject";
 import { unimplementedWarning } from "../utils";
 
 class PopupMenu extends GuiObject {
+  constructor(node, parent) {
+    super(node, parent);
+
+    this.commands = [];
+    this.resolveCmdSelection = null;
```

Comment:
There are private properties right? Can they start with `_`?

Reply:
yup, thanks!

Reply:
oh whoops, they are actually referenced in the rendering. I will rename the resolve function with a `js_` prefix though